tests/components/zha/test_switch.py

```diff
@@ -0,0 +1,67 @@
+"""Test zha switch."""
+from unittest.mock import patch
+from homeassistant.components.zha.core.const import DATA_ZHA
+from homeassistant.const import STATE_ON, STATE_OFF
+from tests.common import mock_coro
+from .common import (
+    async_init_zigpy_device, make_attribute, make_entity_id
+)
+
+SWITCH = 'switch'
+ON = 1
+OFF = 0
+SUCCESS = 0
+
+
+async def test_switch(hass, config_entry, zha_gateway):
+    """Test zha switch platform."""
+    from zigpy.zcl.clusters.general import OnOff
+
+    hass.data[DATA_ZHA][SWITCH] = {}
```

Comment:
When is this normally done?

Reply:
isn't even needed... forgot to remove it.